src/sagas/api.js

```diff
@@ -137,7 +181,14 @@ export function* apiDelete (path, authenticationRequired = true) {
   return yield axios.delete(
     url,
     headers,
-  ).then((res) => _.get(res, 'data', res));
+  ).then((res) => {
```

Comment:
Couldn't this be left alone and update above error handling to simply return `response` 100% of the time?

Reply:
@chris-allen  Yes, but it would require us to update all of our Saga calls in the existing project using this repository.

Reply:
:+1: